fineract-provider/src/main/java/org/apache/fineract/accounting/productaccountmapping/service/ProductToGLAccountMappingWritePlatformServiceImpl.java

```diff
@@ -209,6 +209,16 @@ public void createLoanProductToGLAccountMapping(final Long loanProductId, final
                 this.loanProductToGLAccountMappingHelper.savePaymentChannelToFundSourceMappings(command, element, loanProductId, null);
                 this.loanProductToGLAccountMappingHelper.saveChargesToIncomeAccountMappings(command, element, loanProductId, null);
             break;
+            case CHARGE_OFF_FRAUD:
```

Comment:
We dont need this.... we should save all the configured mapping, but we dont need to differentiate them yet